makefile/scripts/install_python_requirements

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+# Exit script if running from outside of Python Virtual Environment
+[ -z "$VIRTUAL_ENV" ] && echo "This script must be executed from an active Python Virtual Environment, which was not found, exiting..."  && exit 1
+
+# Check if pip CLI is installed in the VENV
+type pip > /dev/null
```

Comment:
Does it make sense to look for pip3 as well?

Reply:
pip is for both pip3 and pip, so fine with first I guess